docs/zkapps/how-zkapps-work.mdx

```diff
@@ -98,11 +101,9 @@ When you deploy to a new Mina address, the Mina Protocol will charge a 1 MINA fe
 <img src="/img/7_zkApps_DeploySmartContract.jpg" width="95%" />
 <br></br>
 
-Developers usually build a UI to allow users to interact with the smart contract.
-
-Typically, this is a static website deployed to a host of the developer’s choosing. We recommend web hosts that offer a global CDN to ensure the best user experience.
+zkApp developers usually build a UI to allow users to interact with the smart contract and then deploy this interactive UI as a static website. To achieve the best user experience, zkApp developers deploy the UI static website to a recommended web host that offers a global content delivery network (CDN).
```

Comment:
The 2nd sentence switches to 3rd person. How about a 2nd sentence of:

> To achieve the best user experience, using a web host that offers a global content delivery network (CDN) is recommended.

(or "best practice" instead of "recommended")

OR

> Choosing a host that offers a global content delivery network (CDN) will ensure the best experience for all of your users.